dao/nft/nft.go

```diff
@@ -94,7 +94,7 @@ func (m *defaultL2NftModel) GetLatestNftIndex() (nftIndex int64, err error) {
 }
 
 func (m *defaultL2NftModel) GetNftsByAccountIndex(accountIndex, limit, offset int64) (nftList []*L2Nft, err error) {
-	dbTx := m.DB.Table(m.table).Where("owner_account_index = ? and deleted_at is NULL", accountIndex).
+	dbTx := m.DB.Table(m.table).Where("owner_account_index = ? and nft_content_hash != ?", accountIndex, "0").
```

Comment:
what‘s the purpose for this change?

Reply:
When nft is withdrawn to L1, we will set nft to empty, in another word, `nft_content_hash` will be set to zero. When user query nft, we shouldn't return the already withdrawn nft.